src/vm/ClrEtwAll.man

```diff
@@ -17,8 +17,8 @@
                              message="$(string.RuntimePublisher.GCKeywordMessage)" symbol="CLR_GC_KEYWORD"/>
                     <keyword name="GCHandleKeyword" mask="0x2"
                              message="$(string.RuntimePublisher.GCHandleKeywordMessage)" symbol="CLR_GCHANDLE_KEYWORD"/>
-                    <keyword name="FusionKeyword" mask="0x4"
-                             message="$(string.RuntimePublisher.FusionKeywordMessage)" symbol="CLR_FUSION_KEYWORD"/>
+                    <keyword name="AssemblyLoaderTracingKeyword" mask="0x4"
```

Comment:
Nit: The "Tracing" in the name seems unnecessary - but I don't feel too strongly about it.